### PR TITLE
Updated AutoCompletions

### PR DESCRIPTION
**Pull request by Monal5031**
Added AutoCompletion in rest of the instructions.
Have tested it my local system too.

### CODE REVIEW

File: LC3-ASM.sublime-completions [hunk 1]
```diff
@@ -7,6 +7,25 @@
             { "trigger": "NOT", "contents": "NOT ${1:Dest}, ${2:Src1}" },
 
             { "trigger": ".FILL", "contents": ".FILL" },
-            { "trigger": ".BLKW", "contents": ".BLKW" }
+            { "trigger": ".BLKW", "contents": ".BLKW" },
+            { "trigger": ".STRINGZ", "contents": ".STRINGZ ${1:String-in-quotes} " },
+
+            { "trigger": "LDR", "contents": "LDR ${1:Dest}, ${2:Src}, ${3:x(OffSet)}"},
```
Comment (aviaryan) [inline LC3-ASM.sublime-completions]: Please make OffSet as `Offset` as it is a valid English word.

File: LC3-ASM.sublime-completions [hunk 2]
```diff
@@ -7,6 +7,25 @@
...
+            { "trigger": "LD", "contents": "LD ${1:Dest}, ${2:LABEL}"},
+            { "trigger": "LDI", "contents": "LDI ${1:Dest}, ${2:LABEL}"},
+            { "trigger": "STR", "contents": "STR ${1:Src}, ${2:Dest}, ${3:x(OffSet)}"},
+            { "trigger": "ST", "contents": "ST ${1:Src}, ${2:LABEL}"},
```
Comment (aviaryan) [inline LC3-ASM.sublime-completions]: Make `LABEL` as `Label` to be consistent with the rest.

File: LC3-ASM.sublime-completions [hunk 3]
```diff
@@ -7,6 +7,25 @@
...
+            { "trigger": "STI", "contents": "STI ${1:Src}, ${2:LABEL}"},
+            { "trigger": "JMP", "contents": "JMP ${1:LABEL"},
+            { "trigger": "BRnzp", "contents": "BRnzp ${1:LABEL}"},
+            { "trigger": "TRAP", "contents": "TRAP ${1:xVector-Address}"},
+            { "trigger": "LEA", "contents": "LEA ${1:R0}, ${2:Src}"},
+            { "trigger": "JSR", "contents": "JSR ${1:LABEL}"},
+            { "trigger": "RET", "contents": "RET"},
+
+            { "trigger": ".END", "contents": ".END"},
+            { "trigger": ".ORIG", "contents": ".ORIG xStart-Address"}
```
Comment (aviaryan) [inline LC3-ASM.sublime-completions]: Missing a `{...}` here.

**Review by aviaryan — CHANGES_REQUESTED** [review]
Please see the comments.

File: LC3-ASM.sublime-completions (hunk 3 above)
Comment (Monal5031) [reply LC3-ASM.sublime-completions]: Sorry didn't get this one.

Comment (aviaryan) [reply LC3-ASM.sublime-completions]: Is this correct? `.ORIG xStart-Address`

Comment (Monal5031) [reply LC3-ASM.sublime-completions]: Yes, we generally write it as `.ORIG x3000`. But yes I got it, `{...}` is needed to show user that it has to be changed as it auto-selects it if we add them

**Comment by Monal5031** [comment]
@aviaryan Please review this one so that I can add the test code too.

**Review by aviaryan — APPROVED** [review]
LGTM

**Pull request merged**